### ISSUE TITLE
Filters.Group: Does not announce state of expanded/collapsed upon activation

### ISSUE THREAD

**Issue by McManning** [osuresearch/ui#94]
Recommendation: As the state changes between expanded and collapsed, change an “aria-expanded” attribute from false to true as appropriate.
